packages/api/test.tap

```diff
@@ -0,0 +1,109 @@
+1..76
```

Comment:
Should the tap file be included in the PR?

Reply:
You are right, it shouldn't, thank you!